data/json/items/magazine/shot.json

```diff
@@ -1,5 +1,15 @@
 [
   {
+    "id": "beltshot",
+    "copy-from": "magazine_belt",
+    "type": "MAGAZINE",
+    "name": "12 gauge ammo belt",
+    "ammo_type": "shot",
+    "capacity": 100,
+    "count": 100,
+    "linkage": "ammolinkshot"
```

Comment:
Not required here but I am starting to wonder if we actually need all the different sizes of linkages versus a generic type. Ping @Rivet-the-Zombie (reality crowd) and @Coolthulhu (gameplay crowd)